### PR TITLE
refactor: patternhub examples to align with figma props

### PR DESCRIPTION
**Pull request by nmerget**
## Proposed changes

Refactor some patternhub examples to align with new properties.

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring (fix on existing components or architectural decisions)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)

### CODE REVIEW

File: showcases/vue-showcase/src/components/radio/Radio.vue [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <script setup lang="ts">
 import DefaultComponent from "../DefaultComponent.vue";
 import defaultComponentVariants from "../../../../shared/radio.json";
-import { DBCheckbox, DBRadio } from "../../../../../output/vue/src";
+import { DBCheckbox, DBInfotext, DBRadio } from "../../../../../output/vue/src";
```
Comment (mfranzke) [inline showcases/vue-showcase/src/components/radio/Radio.vue:4]: @nmerget why are you importing `DBInfotext` in all those files without using it in the code afterwards ?

File: __snapshots__/badge/showcase/chromium/DBBadge-should-match-screenshot-1/DBBadge-should-match-screenshot.png
[the remark this answers is not in the document]
Comment (nmerget) [reply __snapshots__/badge/showcase/chromium/DBBadge-should-match-screenshot-1/DBBadge-should-match-screenshot.png:1]: image

Yes, there was a wrong name but it should be changed on 19th of November already :O

File: showcases/vue-showcase/src/components/radio/Radio.vue (hunk 1 above)
Comment (nmerget) [reply showcases/vue-showcase/src/components/radio/Radio.vue:4]: https://github.com/db-ui/mono/pull/3592/files/29c9e1afbf755c913df5ab8acfcf76d012bfc73e#diff-0bc83bcb39448472a35240dc24527688bf32ebc74e2c420d5ecc9476727d2236R31 <- I use it here

File: showcases/shared/textarea.json [hunk 2]
```diff
@@ -26,7 +26,7 @@
 		]
 	},
 	{
-		"name": "Variant Label",
+		"name": "Variant ",
```
Comment (mfranzke) [inline showcases/shared/textarea.json:29]:
```suggestion
		"name": "Variant",
```

**Comment by mfranzke** [comment]
@nmerget I'm nearly finished with the review.

**Review by mfranzke — APPROVED** [review]

**Pull request merged**